### ISSUE TITLE
Unable to create new character. The error was: null. 

### ISSUE THREAD

**Issue by grymhild** [BahamutDragon/pcgen#973]
Fresh install of Windows version of PCGen 6.08.00 RC8 
BahamutDragon Branch: 6,08

**Comment by Crosline** [comment]
It is probably because of the data sets... Try to load another set and you will see it is working... (pathfinder wasn't working for me but 5e)

**Comment by fatelfking** [comment]
Are all the Bahamut DL compatible with the new PCGEN?

**Comment by fatelfking** [comment]
![image](https://user-images.githubusercontent.com/63133809/78418059-b5312680-7606-11ea-8b9b-62600e35b15d.png)

**Comment by BahamutDragon** [comment]
Log? Sets loaded? Need some info to proceed to correct.

**Comment by BahamutDragon** [comment]
6.08 is designed to be compatible with PCGen 6.08; Master is designed to be compatible with the 6.09 development.

**Comment by fatelfking** [comment]
I've always used your files and never had a problem, I'm going to bet I'm doing something wrong lol. one of my issues is, the last time I coded was about 7 years ago hahaha and I just started about 4 days ago lmao. I read your instructions and I'm following them step by step and I keep getting that same error.  If this helps, after I unzip when I go and dump everything into each pertaining folder it doesn't override anything which should mean I'm doing it correct, right? now after I dump everything if I go back to the original PCGEN source I still get the same issue.
